api/v1/requests/RSVPRequest.js

```diff
@@ -0,0 +1,19 @@
+var Request = require('./Request');
+var rsvp = require('../utils/rsvp');
+
+var bodyRequired = ['attendeeAttendance'];
+var bodyValidations = {
+    'attendeeAttendance': ['required', 'string', rsvp.verifyAttendanceReply]
```

Comment:
I'm just pulling the attendeeId from the request user I didn't think it made sense to have to include that in the request.